packages/core/src/server/service.ts

```diff
@@ -195,14 +199,18 @@ export class ServerService {
         case "POST":
           return graphqlMiddleware(request, response, next);
         case "GET": {
+          const host = request.get("host");
+          if (!host) {
+            return response.status(400).send("No host header provided");
```

Comment:
Do you know if there are any valid cases where this would happen?

Reply:
I don't but `host` type is `string | undefined` so I added this check to make ts happy